scala/lambdas/ingest-mapper/src/main/scala/uk/gov/nationalarchives/ingestmapper/DiscoveryService.scala

```diff
@@ -28,7 +28,7 @@ trait DiscoveryService[F[_]] {
 
 }
 object DiscoveryService {
-  case class DiscoveryScopeContent(description: String)
+  case class DiscoveryScopeContent(description: Option[String])
```

Comment:
Maybe potentialDescription?

Reply:
Sorry, the tests were failing because we're decoding the json from discovery to this class so it has to stay as description.